src/sdk/models/metadataFilters.ts

```diff
@@ -0,0 +1,81 @@
+export enum MetadataConjunctionOperator {
+	And = 'and',
+	Or = 'or'
+}
+
+export enum MetadataPropertyFilterOperator {
+	eq = 'Equals',
```

Comment:
These should actually be reversed (i.e. `Equals = 'eq'`).

Reply:
Should I update the enum member name as well then (change 'Less Than' or LessThan) or leave as is:

export enum MetadataPropertyFilterOperator {
	Equals = 'eq',
	'Not Equal'= 'ne',
	'Less Than' = 'lt',
	'Less Than Equal To' = 'le',
	'Greater Than' = 'gt',
	'Greater Than Equal To' = 'ge',
	'Starts With' = 'starts_with',
	'Does Not Start With' = 'not_starts_with',
	'Ends With' = 'ends_with',
	'Does Not End With' = 'not_ends_with',
	Contains = 'contains',
	'Does Not Contain' = 'not_contains',
	In = 'in',
	'Not In' = 'not_in',
	Empty = 'empty',
	'Not Empty' = 'not_empty',
}

Reply:
Updated